ui/component/app/view.jsx

```diff
@@ -257,7 +258,18 @@ function App(props: Props) {
       };
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [hasVerifiedEmail, syncEnabled, checkSync]);
+  }, [checkSync, readyForSync, syncEnabled]);
+
+  // Currently we know someone is logging in or not when we get their user object {}
+  // We'll use this to determine when it's time to pull preferences
+  // This will no longer work if desktop users no longer get a user object from lbryinc
+  useEffect(() => {
+    if (user) {
+      if (typeof user === 'object' || typeof user === 'string') {
```

Comment:
When would `user` be a string?

Reply:
It's a bug in the code right now. I'll just make it if (user).